main.py

```diff
@@ -770,6 +775,12 @@ def clean_up_db(self):
             if len(verification[email]['verified']) == 1 and len(verification[email]['unverified']) >= 1:
                 db.delete(verification[email]['unverified'])
 
+class CronHandler(BaseHandler):
```

Comment:
Might want to make this into a UpdateLunchHandler, and skip the 'job' argument. We took a different approach for /admin, but I tend to think this approach would be simpler.